Source/CesiumEditor/Private/PCH.h

```diff
@@ -1,29 +0,0 @@
-// Copyright 2020-2024 CesiumGS, Inc. and Contributors
```

Comment:
PCH stands for "precompiled header". It's a header file that is included by all (or at least most) of the source files with the intention of making builds go faster by only processing big, complicated, and very commonly used headers once per project instead of once per source file. The downside is that it makes every object file bigger if the PCH includes headers that some source files otherwise wouldn't need. And also if any of the headers included by the precompiled header change, you get to recompile everything. I removed it because it wasn't really doing us any good.